### PR TITLE
set warning severity for phpcs

### PR DESCRIPTION
**Pull request by tvestelind**
I needed to be able to control how much warnings to show. I hope this is the correct way of doing that.

### CODE REVIEW

File: autoload/neomake/makers/ft/php.vim [hunk 1]
```diff
@@ -26,6 +26,11 @@ function! neomake#makers#ft#php#phpcs() abort
         call add(l:args, '--standard=' . expand(g:neomake_php_phpcs_args_standard))
     endif
 
+    "Add warning severity argument if one is set.
+    if exists('g:neomake_php_phpcs_args_warning_severity')
+        call add(l:args, '--warning-severity=' . expand(g:neomake_php_phpcs_args_warning_severity))
+    endif
+
```
Comment (blueyed) [inline autoload/neomake/makers/ft/php.vim]: You could just do `let neomake_php_phpcs_arg = neomake#makers#ft#php#phpcs().args += ['--warning-severity=whatever']`.
I see that there is `g:neomake_php_phpcs_args_standard` already, and it is not necessary in the same way.

Please report back if that works for you.

(there is some idea about having a `post` setting for args, see https://github.com/neomake/neomake/issues/972#issuecomment-275800406)

Comment (tvestelind) [reply autoload/neomake/makers/ft/php.vim]: Hi,
I tried to add it to my config file but that didn't work. Is that how you mean that I should do it?
The error that I got is:
```
E15: Invalid expression: = ['--warning-severity=0']
E15: Invalid expression: neomake#makers#ft#php#phpcs().args += ['--warning-severity=0']
```

Comment (blueyed) [reply autoload/neomake/makers/ft/php.vim]: Sorry, should be `+` only. Fixed above.

Comment (blueyed) [reply autoload/neomake/makers/ft/php.vim]: @tvestelind 
Does this work for you?

Comment (tvestelind) [reply autoload/neomake/makers/ft/php.vim]: Hi,
Sorry for taking so long.
Yea, it seems to work. Is that they way you prefer configuring tools?

**Pull request closed without merging**